app/helpers/container_route_helper/textual_summary.rb

```diff
@@ -28,4 +28,38 @@ def textual_creation_timestamp
   def textual_resource_version
     {:label => "Resource Version", :value => @record.resource_version}
   end
+
+  def textual_ems
```

Comment:
Remove entirely this method (it has been unified somewhere else so you can just drop it).